testing/functional/tests/cypress.json

```diff
@@ -16,5 +16,6 @@
   },
   "video": false,
   "chromeWebSecurity": false,
-  "defaultCommandTimeout": 10000
+  "defaultCommandTimeout": 10000,
+  "numTestsKeptInMemory": 0
```

Comment:
Was it changed due to high memory consumption?

Reply:
If the test takes more than 200 seconds in Cypress, the following screen appears:

'Aw, Snap! Something went wrong while displaying this webpage. Learn More / Reload'
![image](https://user-images.githubusercontent.com/98920571/162979125-ed98fece-21c6-4750-ab72-2faf8a9b6972.png)

By adding, in cypress.json
{
"numTestsKeptInMemory": 0,
}
Browser is not crashing.